.clang-tidy

```diff
@@ -0,0 +1,10 @@
+Checks: "bugprone-*,performance-*,readability-*,modernize-*,hicpp-*,cppcoreguidelines-*,clang-analyzer-*,-readability-braces-around-statements,-hicpp-braces-around-statements,-cppcoreguidelines-non-private-member-variables-in-classes"
```

Comment:
Are these reasonable checks?  Some, like [`modernize-use-trailing-return-type`](https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-trailing-return-type.html) can be a bit much at first.

Reply:
What checks would be good to keep?

Reply:
Fixed in 13b6883.

Reply:
Unresolved to confirm lint checks.

Reply:
I found [website](https://clang.llvm.org/extra/clang-tidy/checks/list.html) that listed a lot of the lint checks, but there were hundreds. What are the bare minimum checks I should have?

Reply:
I think we might be close; I think it might just be the trailing return type thing:
`bugprone-*,performance-*,readability-*,modernize-*,hicpp-*,cppcoreguidelines-*,clang-analyzer-*,-modernize-use-trailing-return-type,-readability-braces-around-statements,-hicpp-braces-around-statements,-cppcoreguidelines-non-private-member-variables-in-classes`

Reply:
Fixed in 5a21c06.